docs/sources/send-data/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+menuTitle: Send data
+title: Send log data to Loki
+description: Grafana Loki clients
+weight: 600
+---
+# Send log data to Loki
+
+You can use the following clients to send logs to Grafana Loki:
+
+- [Grafana Agent]({{< relref "/docs/agent" >}}) 
+- [Promtail]({{< relref "../clients/promtail" >}})
+-- [Promtail on AWS EC2]({{< relref "./aws/ec2" >}})
+-- [Promtail on AWS ECS]({{< relref "./aws/ecs" >}})
+-- [Promtail on AWS EKS]({{< relref "./aws/eks" >}})
+- [Docker Driver]({{< relref "./docker-driver" >}})
+- [Fluentd]({{< relref "./fluentd" >}})
+- [Fluent Bit]({{< relref "./fluentbit" >}})
+- [Logstash]({{< relref "./logstash" >}})
+- [Lambda Promtail]({{< relref "./lambda-promtail" >}})
+
+There are also a number of third-party clients, for a list see [Unofficial clients](#unofficial-clients).
+
+The [xk6-loki extension](https://github.com/grafana/xk6-loki) permits [load testing Loki]({{< relref "./k6" >}}).
+
+## Picking a client
+
+While all clients can be used simultaneously to cover multiple use cases, which
+client is initially picked to send logs depends on your use case.
+
+### Promtail
+
+Promtail is the client of choice when you're running Kubernetes, as you can
+configure it to automatically scrape logs from pods running on the same node
+that Promtail runs on. Promtail and Prometheus running together in Kubernetes
+enables powerful debugging: if Prometheus and Promtail use the same labels,
+users can use tools like Grafana to switch between metrics and logs based on the
+label set.
```

Comment:
```suggestion
Promtail is the client of choice when you're running Kubernetes, as you can configure it to automatically scrape logs from Pods running on the same node that Promtail runs on. Promtail and Prometheus running together in Kubernetes enables powerful debugging: if Prometheus and Promtail use the same labels, users can use tools like Grafana to switch between metrics and logs based on the label set.
```

https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/#kubernetes-objects says we should capitalize Kubernetes objects like Pods to be consistent with Kubernetes documentation.